VehicleInformationMsg info
---
VehicleStatusMsg vehicle_status
int16 pms_id
bool load_vehicle
